src/Uno.UWP/Globalization/NumberFormatting/NumeralSystemTranslator.cs

```diff
@@ -0,0 +1,169 @@
+using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Linq;
+using Uno.Globalization.NumberFormatting;
+
+namespace Windows.Globalization.NumberFormatting
+{
+	public partial class NumeralSystemTranslator
+	{
+		private string _numeralSystem;
+
+		public NumeralSystemTranslator() : this(new string[] { "en-US" })
```

Comment:
Wouldn't this allocate unnecessarily? Consider extracting this as an `ImmutableArray` static field.

Reply:
Nice comment. Immutable Array is not supported by Net461 do you agree with private static readonly array?

Reply:
Readonly array should be enough, but I think even the current solution is good enough. I don't see a scenario where there would be thousands of instances of `NumeralSystemTranslator` and the allocation would be a problem 🤔 .

Reply:
Extracting to a readonly static field could be enough indeed, even if the number of instances is likely not going to be very large.